samples/generic-base/user.env

```diff
@@ -20,10 +20,16 @@
 # placement strategy
 # NODE_PLACEMENT_ENABLE=false
 
-
 # Namespace of NGINX ingress controller (if applicable)
 # NGINX_NS=ingress-nginx
 
 # Name of NGINX ingress controller (if applicable)
 # NGINX_SVCNAME=ingress-nginx-controller
 
+# OpenShift-specific options
+# Enable path-based routes (URLs will end in /grafana, /kibana, etc.)
+# OPENSHIFT_PATH_ROUTES=false
+# Override the base route hostname used for monitoring and logging
+# Note that the actual hostname used will have the name of the service
+# and namespace prepended (e.g. v4m-grafana-$MON_NS.$OPENSHIFT_ROUTE_DOMAIN)
+# OPENSHIFT_ROUTE_DOMAIN
```

Comment:
See comment above about providing a sample value or an obvious non-sense value.  As written, it is not clear that this is a value the user can/should change.

Reply:
Updated